### DISCUSSION TITLE
Deleting recrods limited to one page

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14864] [Feature improvement]
I can not accumulate records for mass deletion page by page. If I select all records in current screen and move to another set of records it clearing marked records from the previous page. Please see attached.

Normal behavior should be that selected records are kept selected,  even we changing page. This will allow for less annoying mass actions on records :) 
![deleting](https://user-images.githubusercontent.com/84110093/123185972-508b1180-d4c1-11eb-97f3-c7308514b2b6.gif)

**Comment by apcloic** [comment]
@s2b-git 
Hi,
Deleting is not limited to one page, you also have the option to select all records across all page.
You could always use a custom filter and some criterias to select what you want and then mass delete all.

**Comment by s2b-git** [comment]
Maybe I am doing something wrong, but look at this: in developer version in documents module there was 42 records. Each screen contains 15 records. I selected all records in the first screen, moved to another and selected another set of records from second screen. Than I used "Mass delete". The number of records what left over in the module is now 22. 
If I do quick math it should be like that: 42-30=12 (not 22)
So I believe there is some invisible glitch ...?

**Comment by mariuszkrzaczkowski** [comment]
https://user-images.githubusercontent.com/10198654/123908047-292bbd00-d977-11eb-9b85-b794149ecdea.mp4

**Comment by garryatbrighton** [comment]
Agree, it is a performance glitch. Moving across pages erasing selection of records. If I navigate across 1000 records (50 records per screen) I cannot select few of them on each  view, as moving forward or backward erasing selection. It could be a big deal if you want manually and efficiently execute some action of records.
